src/main/java/com/dev/cinema/model/dto/request/OrderRequestDto.java

```diff
@@ -0,0 +1,33 @@
+package com.dev.cinema.model.dto.request;
+
+import java.util.List;
+
+public class OrderRequestDto {
```

Comment:
Do you use it anywhere?

Reply:
this I need for `public Order getEntity(OrderRequestDto orderRequestDto)` that I don't use)

Reply:
That's the point =) You create classes that you don't use just because you have specific interface. SOLID - I - Interface Segregation